app/code/community/Bolt/Boltpay/Helper/ConfigTrait.php

```diff
@@ -236,4 +236,38 @@ public static function getBoltPluginVersion()
 
         return null;
     }
+
+    /**
+     * Returns message to be displayed to customer if attempting to checkout with order lower than minimum amount
+     *
+     * @return string
+     */
+    public function getMinOrderDescriptionMessage()
+    {
+        return Mage::getStoreConfig('sales/minimum_order/description')
```

Comment:
Might be worth a minor refactor here like this:

return Mage::getStoreConfig('sales/minimum_order/description') ?:     Mage::helper('checkout')->__('Minimum order amount is %s',$this->getMinOrderAmountInStoreCurrency());

Not required and more just because I was trying to find something I could be helpful with :)